fotoapparat/src/main/java/io/fotoapparat/hardware/v1/capabilities/CapabilitiesFactory.java

```diff
@@ -86,4 +88,52 @@ public Capabilities fromParameters(Camera.Parameters parameters) {
 		return result;
 	}
 
+	private DiscreetRange extractSupportsSensorSensitivityValue(Camera.Parameters parameters) {
```

Comment:
This method does too much. Normally method should not be longer than 20 lines. I suggest to extract a class and make it a dependency of `CapabilitiesFactory`